tutorials/2d/custom_drawing_in_2d.rst

```diff
@@ -384,7 +384,7 @@ calls _draw(). This way, you can control when you want to refresh the frame.
         _angleFrom += _rotationAngle;
         _angleTo += _rotationAngle;
 
-        // We only wrap angles if both of them are bigger than 360
+        // We only wrap angles if each of them is bigger than 360.
```

Comment:
Here and above: IMHO "both" is better than "each", as it emphasizes that both conditions should be met, and not that something is done for each angle.
Potentially, "if" can be changed to "when".

Reply:
@bojidar-bg : I disagree with your first assessment; think of "each" as a filter function that takes 2 arguments and checks whether  **each** of them meets a given condition, and then performs a logical conjunction on those intermediary results.
Either way, I've chosen to go the "when" route.
All in all, I think the quality of your remarks has improved considerably since the last time I "saw" you. Keep at it! :sunglasses: